deployments/stacks/LocalKindStack.ts

```diff
@@ -0,0 +1,110 @@
+import { Construct } from "constructs";
+import * as cdktf from "cdktf";
+import * as kubernetes from "@cdktf/provider-kubernetes";
+import * as kubectl from "../.gen/providers/kubectl";
+import * as helm from "@cdktf/provider-helm";
+import * as kind from "../.gen/providers/kind";
+import * as http from "@cdktf/provider-http";
+import { CarbyneStack } from "../constructs/CarbyneStack";
+import { VPC } from "../constructs/VPC";
+
+export interface LocalKindConfig {
+  num_clusters?: number;
+}
+
+export default class LocalKindStack extends cdktf.TerraformStack {
+  constructor(scope: Construct, id: string, config: LocalKindConfig) {
+    super(scope, id);
+
+    const kind_provider = new kind.provider.KindProvider(this, "kind-provider");
+
+    const num_clusters =
```

Comment:
The whole logic does not work for >= 2 VCPs (e.g. MAC key shares. Should we remove the option to configure it or live on the edge?

Reply:
it is now hard-coded.  the idea was to port the fake key generation from MPSPDZ:  https://github.com/data61/MP-SPDZ/blob/2813c0ef0fa9ca48ced3f70b7081c071f0bd28fd/Protocols/fake-stuff.hpp#L451

Reply:
@jaredweinfurtner I have no problem doing this in a first run, but could you please create an issue to allow more than 2 players in the future.

Reply:
done: https://github.com/carbynestack/carbynestack/issues/90